libbeat/tests/system/beat/beat.py

```diff
@@ -198,7 +198,7 @@ def render_config_template(self, template_name=None,
         if template_name is None:
             template_name = self.beat_name
 
-        template_path = "./tests/system/config/" + template_name + ".yml.j2"
+        template_path = os.path.join("./tests/system/config", template_name + ".yml.j2")
```

Comment:
Thanks fo rthe cleanup

Reply:
Interestingly this was working before with "only" forward slashes but now that we start mixing forward and backward slashes, it breaks on windows. Will open a follow up PR.